minitest-reporters.gemspec

```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.summary     = %q{Create customizable Minitest output formats}
   s.description = %q{Death to haphazard monkey-patching! Extend Minitest through simple hooks.}
 
+  s.required_ruby_version = '>= 1.9.3'
```

Comment:
@os97673 I just added the test on Ruby 1.8.7 in the PR but tests are failing, do we want to fix code to support 1.8.7?